### PR TITLE
Improve docs for creating new wraps.

### PR DESCRIPTION
**Pull request by jpakkane**
I got some IRL comments that the existing docs are a bit confusing so I expanded them a bit.

### CODE REVIEW

**Comment by dcbaker** [comment]
This looks good, maybe replace emacs with $EDITOR though?

File: docs/markdown/Adding-new-projects-to-wrapdb.md [hunk 1]
```diff
@@ -72,35 +72,104 @@ will be posted in a comment to this issue.
...
-First you need to fork the repository to your own page.
-Then you can create the first Wrap commit that usually looks something like this.
+Setting up the contents might seem a bit counterintuitive at
+first. Just remember that the outcome needs to have one (and only one)
+commit that has all the build definition files (i.e. `meson.build` and
+`meson_options.txt` files) and _nothing else_. It is good practice to
+have this commit in a branch whose name matches the release as
+described above.
+
+First you need to fork the repository to your own page using Github's
```
Comment (legeana) [inline docs/markdown/Adding-new-projects-to-wrapdb.md:84]: "GitHub" so that it's consistent with the text below.

**Review by legeana — CHANGES_REQUESTED** [review]
LGTM

**Review by dcbaker — APPROVED** [review]
Lgtm

**Pull request merged**